flow-typed/npm/redux_v3.x.x.js

```diff
@@ -10,26 +10,33 @@ declare module 'redux' {
 
   */
 
-  declare type Dispatch<A: { type: $Subtype<string> }> = (action: A) => A;
+  // perf.html - Add thunks to the dispatch definitions.
+  declare type GetState<S> = () => S;
+  declare type ThunkAction<S, A, R> = (dispatch: Dispatch<S, A>, GetState<S>) => R;
+  declare type PromiseAction<R> = { type: string, payload: Promise<R> };
+  declare type ThunkDispatch<S, A> = <R>(action: ThunkAction<S, A, R>) => R;
+  declare type PromiseDispatch = <R>(action: PromiseAction<R>) => Promise<R>;
```

Comment:
note we don't have the Promise-version of actions and dispatch in our code currently, it needs another middleware. So maybe we should remove these from our types for now, and add it if we ever add redux-promise in the future ?